code/modules/paperwork/paper.dm

```diff
@@ -163,6 +175,15 @@
 		update_icon()
 		return
 
+	if (user.a_intent == I_HELP && old_name && icon_state == "paper_swan")
```

Comment:
honestly just make it `(icon_state == "paper_plane" || icon_state == "paper_swan")` in the conditional above and delete the rest of this since it's two blocks of code that do the same thing otherwise

Reply:
Donezo